dist/main.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+const core = require("@actions/core");
+const plivo = require("plivo");
+async function run() {
+    const from = core.getInput("fromPhoneNumber");
+    const to = core.getInput("toPhoneNumber");
+    const message = core.getInput("message");
+    core.debug(from, to, message);
+    const auth_id = core.getInput("PLIVO_AUTH_ID") || process.env.PLIVO_AUTH_ID;
+    const auth_token = core.getInput("PLIVO_AUTH_TOKEN") || process.env.PLIVO_AUTH_TOKEN;
+    var client = new plivo.Client(auth_id, auth_token);
+    var response = client.messages.create(from, to, message).then(function (response) {
```

Comment:
not sure would there be anything in response since we are catching the response via promises.

Reply:
I'm using the variable to resolve to transmit the error to catch block in async